comp/venue/batteries.yaml

```
@@ -0,0 +1,37 @@
+summary: Organise battery management for the $SRYYYY competition
+
+priority: should
```

Robots can't do much without batteries

```suggestion
priority: must
```